src/main/scala/zio/prelude/Assertions.scala

```diff
@@ -13,7 +13,7 @@ trait Assertions {
   /**
    * Makes a new assertion that requires a value equal the specified value.
    */
-  def equalTo[A: Equal](expected: A): Assertion[A] =
+  def isEqualTo[A: Equal](expected: A): Assertion[A] =
```

Comment:
I think it would be nice to keep this one as `equalTo` for symmetry with the default `equalTo` assertion in ZIO Test.

Reply:
done :heavy_check_mark: